drivers/gles2/rasterizer_gles2.cpp

```diff
@@ -261,7 +261,17 @@ void RasterizerGLES2::initialize() {
 #endif // GLES_OVER_GL
 #endif // CAN_DEBUG
 
-	print_line("OpenGL ES 2.0 Renderer: " + VisualServer::get_singleton()->get_video_adapter_name());
+	bool using_batching = false;
+	if (Engine::get_singleton()->is_editor_hint()) {
+		if (GLOBAL_GET("rendering/batching/options/use_batching_in_editor")) {
+			using_batching = true;
+		}
+	} else {
+		if (GLOBAL_GET("rendering/batching/options/use_batching")) {
+			using_batching = true;
+		}
+	}
```

Comment:
```suggestion
	if (Engine::get_singleton()->is_editor_hint()) {
		using_batching = GLOBAL_GET("rendering/batching/options/use_batching_in_editor");
	} else {
		using_batching = GLOBAL_GET("rendering/batching/options/use_batching");
	}
```

But I think it's best to keep this in the actual batcher code, and just make it `print_verbose` like the rest. I don't think it's primordial information to have on the same line as the renderer, and most importantly, this only checks the project settings while the batcher actually provides the state of what is getting initialized. Could be that project settings say one thing but the batcher code overrides the setting and does something else. (I don't think it does, but in theory it would be possible, just like you can have GLES3 configured in project settings and end up running on GLES2 if you have fallback enabled and no GLES3 support.)